lib/news.arc

```diff
@@ -390,22 +390,22 @@
 
 (defopr favicon.ico req favicon-url*)
 
-(def gen-css-url ()
-  (tag (link "rel" "stylesheet" "type" "text/css" "href" "news.css")))
-
 (mac npage (title . body)
-  `(tag html
-     (tag head
-       (gen-css-url)
-       (tag (link "rel" "shortcut icon" "href" favicon-url*))
-       (tag (meta "name" "viewport" "content" "width=device-width"))
-       (tag script (pr votejs*))
-       (tag title (pr ,title)))
-     (tag body
-       (center
-         (tag (table border 0 cellpadding 0 cellspacing 0 width "85%"
-                     bgcolor sand)
-           ,@body)))))
+  `(do 
+;   (prn "<!DOCTYPE html>")
```

Comment:
Yeah this is very weird. I tried a different doctypes, and I tried to figure out what doctype the browser currently things it is (`null`. Does that mean quirks mode?)

The CSS file is loading fine. So I don't understand why it isn't being used.

Can you add a comment about why it's commented out?

Reply:
Whenever I tried using a doctype it broke stylesheet rendering, but I didn't know if that was just me or something to do with the HTML, so I thought it was best to err on the side of caution and let other people test it out.